tutorminio/patches/caddyfile

```diff
@@ -1,4 +1,4 @@
 # MinIO
-{{ MINIO_HOST }}{% if not ENABLE_HTTPS %}:80{% endif %} {
-    reverse_proxy nginx:80
+{{ MINIO_HOST }}{$default_site_port} {
+    import proxy "minio:9001"
```

Comment:
Wait... What is the difference between the server and the console ports? If I understand correctly, the console port is for accessing the web UI, right? The problem is that other services, such as the LMS/CMS, need to be able to access the server from the https://minio.mylms.com address. So the MINIO_HOST should point to the minio server, and not the console.

What this means is that we might need to expose the MinIO console at a different web host. Something like:

```
{{ MINIO_CONSOLE_HOST }}{$default_site_port} {
    import proxy "minio:9000"
{{ MINIO_CONSOLE_HOST }}{$default_site_port} {
    import proxy "minio:9001"
}
```

Do you understand my point?